aikiblog/templates/add_techniques.html

```diff
@@ -50,7 +50,7 @@
 			<td>{{ technique.date|date:" d.m.Y " }}&nbsp;&nbsp;</td>
 			<td>{{ technique.stand }}&nbsp;&nbsp;</td>
 			<td>{{ technique.attack }}&nbsp;&nbsp;</td>
-			<td>{{ technique.technique }}&nbsp;&nbsp;</td>
+			<td><a href="{% url "technique-list" technique.technique.slug %}">{{ technique.technique }}</a>&nbsp;&nbsp;</td>
```

Comment:
poprawiłem na techtrens. Jeśli chodzi o myślnik, to już teraz tak zostawię i będę podążał tą konwencją nazywania. Przy nowym prodżekcie już będę robił z podkreślnikiem, bo teraz to za dużo zmieniania, a jest czytelne przy odnoszeniu się do urla w templacie